staging/src/k8s.io/kubectl/pkg/cmd/cp/cp.go

```diff
@@ -322,7 +322,7 @@ func (o *CopyOptions) copyFromPod(src, dest fileSpec) error {
 		cmdutil.CheckErr(err)
 	}()
 	prefix := getPrefix(src.File)
-	prefix = path.Clean(prefix)
+	prefix = filepath.Clean(prefix)
```

Comment:
if we're copying from a pod, isn't src.File a linux path? if so, will filepath.Clean do the right thing when `cp` is run on windows?

Reply:
We want to ensure that the client-side cleaning is relevant for their particular system. Actually this should do both unix and windows cleaning, I'll probably replace all windows and proceed with processing only linux-style separator.